foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -212,7 +238,41 @@ Registering the extension with Core, sends Experience Platform SDK events to an
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+Registering the extension with Core sends Experience Platform SDK events to an active Project Griffon session. To start using the extension library, you must first register the extension with the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension.
+
+#### Objective-C
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+
+    NSArray *extensionsToRegister = @[AEPMobileLifecycle.class, AEPMobileAssurance.class];
```

Comment:
Can we use something like NSArray *extensionsToRegister = @[AEPMobileAssurance.class, ...];